cdap-docs/reference-manual/source/http-restful-api/workflow.rst

```diff
@@ -11,10 +11,12 @@ Workflow HTTP RESTful API
 
 .. highlight:: console
 
+Workflow Tokens
+===============
+
```

Comment:
Short answer: It doesn't actually matter, as the formatted HTML is controlled by the CSS of the theme we specify and use with Sphinx. 

Long answer: JJ (me) is inconsistent with his use of spacing, and should (and will) try to do a better job in the future. I will add something to our rst style guide about this. I think an appropriate approach would be before every major title (those are the first ones, usually indicated with "=" underlining), that there be two spaces before. (An exception could possibly be made if there already something that has the same affect, such as an anchor or highlight.)  And one space before the next title level (usually indicated with "-" underlining). After the underlining, text could be tight, which would keep the file size (as rendered in an editor) shorter and quicker to navigate. I will adjust this file to match.